librad/src/git/storage/fetcher.rs

```diff
@@ -262,6 +262,20 @@ pub mod error {
         #[error(transparent)]
         Pool(#[from] super::PoolError),
     }
+
+    #[derive(Debug, Error)]
+    pub enum FetchError {
+        #[error("Fetch limit {limit} exceeded. Amount fetched: {amount_fetched} from {remote}. Fetch specs: {fetch_specs:?}, refspecs: {refspecs:?}")]
```

Comment:
I'm not sure it is a good default to dump the specs in the `Display` impl -- these can be very large, and most likely only provide value when at least `DEBUG` is enabled. Since the `Debug` impl will amount to the same information, maybe leave it to the call site to log it out at the appropriate level?

Reply:
Good shout, fix pushed